relm4/Cargo.toml

```diff
@@ -15,12 +15,22 @@ documentation = "https://relm4.org/docs/stable/relm4/"
 keywords = ["gui", "gtk", "gtk4", "elm"]
 categories = ["gui"]
 
+[package.metadata.docs.rs]
+all-features = true
+cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples=examples"]
+
 [features]
 default = []
+# The dox feature can be used for building the docs without the dependencies and requires Rust Nightly
+dox = ["gtk/dox", "adw/dox", "panel/dox"]
 libadwaita = ["adw"]
 libpanel = ["panel"]
 macros = ["relm4-macros"]
-all = ["libadwaita", "macros"]
+all = [
+    "libadwaita",
+    "macros",
+    # dox is not part of the 'all' feature, because it is only used for generating the documentation
+]
```

Comment:
Since "all" doesn't include libpanel and therefore the name is probably rather confusing, I suggest the following:

+ Make "macros" a default feature
+ Remove the "all" feature
+ Update the feature description in the README accordingly

Reply:
Okay, sure